docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Dockerfile for awesomeWM.
```

Comment:
Yes, makes sense, although I'd prefer shorter names.. ;)
I think for now we can have it at the top level though for our main one.